packages/cli/src/services/checkly-config-loader.ts

```diff
@@ -74,6 +74,10 @@ function loadFile (file: string) {
   }
 }
 
+function isString (obj: any) {
+  return (Object.prototype.toString.call(obj) === '[object String]')
```

Comment:
Why doesn't `instanceof` work?

Reply:
Short answer, it is javascript. It doesn't consider const example = "string" as a member of the String class